### ISSUE TITLE
Eyelid pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9864]
**Preferred term label:**
Eyelid pain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hordeolum

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034808
Eyelid pain
